master/pkg/model/task.go

```diff
@@ -115,6 +115,8 @@ type Allocation struct {
 	// ProxyAddress stores the explicitly provided task-provided proxy address for resource
 	// managers that do not supply us with it. Comes from `determined.exec.prep_container --proxy`.
 	ProxyAddress *string `db:"proxy_address" bun:"proxy_address"`
+	ExitReason   string  `db:"exit_reason" bun:"exit_reason"`
```

Comment:
I think we should probably make ExitReason / StatusCode a ptr so that we know they are "optional" and won't be always filled in

Reply:
Done!